Automattic/vip-go-mu-plugins
# Jetpack 6.3.3

https://jetpack.com/2018/07/30/jetpack-6-3-3-facebook-publicize/

> Starting August 1, 2018, a change to Facebook’s API means that third-party tools can no longer share posts automatically to Facebook Profiles. This includes Publicize, our feature that connects your site to major social media platforms like Facebook.
>
> As a result, this release includes both the change to the Publicize feature, and a notice that is designed to help you with next steps

More specifics on the changes here: https://github.com/Automattic/jetpack/milestone/107

## To Test

- Apply patch
- Login to wp-admin
- Make sure Jetpack shows version at 6.3.3
